firebase-firestore/src/test/java/com/google/firebase/firestore/model/FieldValueTest.java

```diff
@@ -233,46 +223,45 @@ public void testDeletesMultipleNewFields() {
   @Test
   public void testValueEquality() {
     new EqualsTester()
-        .addEqualityGroup(wrap(true), BooleanValue.valueOf(true))
-        .addEqualityGroup(wrap(false), BooleanValue.valueOf(false))
-        .addEqualityGroup(wrap(null), NullValue.nullValue())
+        .addEqualityGroup(wrap(true), wrap(true))
```

Comment:
The test verifies that the equality works across different instances of the same object.  Since I agree that `wrap(valueOf(false))` is more visually pleasing (and makes the test's purpose less questionable), I employed your suggestion.